datasets/hk/po/crawler.py

```diff
@@ -0,0 +1,76 @@
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import OccupancyStatus, categorise
+
+from normality import collapse_spaces
+from xml.etree import ElementTree
+
+
+def get_element_text(doc: ElementTree, xpath_value: str, to_remove=[], position=0):
+    element_tag = doc.xpath(xpath_value)
+    element_text = element_tag[position].text_content() if element_tag else ""
+
+    for string in to_remove:
+        element_text = element_text.replace(string, "")
+
+    return collapse_spaces(element_text.strip())
+
+
+def crawl_members(context: Context, section: str, elem: ElementTree):
+
+    url = elem.get("href")
+    url = context.data_url.replace("index.htm", url)
+
+    doc = context.fetch_html(url, cache_days=1)
+
+    member_header = get_element_text(doc, '//h1[contains(@class,"title")]')
+    member_desc = get_element_text(doc, '//section[@class="blockItem"]')
+
+    person_name = member_header.split(",")[0]
+    person_title = person_name.split()[0]
+    person_name = " ".join(person_name.split()[1:])
+
+    position_name = member_header.split(",")[-1]
+    position_name = f"{section} - {position_name.strip()}"
```

Comment:
mmm I think `Secretary for Culture, Sports and Tourism` is a perfect position name.
I think `Directors of Bureaux - Secretary for Culture, Sports and Tourism` is more than we need. So let's rather not prefix the position name with it.

You could stick `Directors of Bureaux` in to the position description while you have it at hand.